config/environments/production.rb

```diff
@@ -44,7 +44,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = false
```

Comment:
Se que de momento queremos deshabilitar esta configuracion  para poder trabajar en el servidor de pruebas sin forzar las peticiones a ir sobre un SSL.

Me parece bien el cambio, pero tambien tengo una propuesta 😄 .

Que opinas de hacer que esta configuracion la podamos controlar a traves de una variable de ambiente?

La idea es:

1. Que no se nos pase volver a prender esta funcion cundo nos pacemos al servidor de produccion
2. Evitar tener que hacer un cambio de codigo cuando lo movamos
3. Que ambos ambientes funcionen con el mismo codigo.

```suggestion
  config.force_ssl = ENV.fetch("FORCE_SSL")
```

Reply:
¿Con modificarlo en production.rb será suficiente o de plano también en initializers/decidim.rb?

Reply:
En `production.rb` es suficiente. En el otro fichero podemos dejarlo comentado como estaba 🙌🏽